brave/brave-ios
# Fix #2564: Update Rewards lib to include ads crash fix

## Summary of Changes


This pull request fixes #2564 

Rewards lib change: https://github.com/brave/brave-core/pull/5614

## Submitter Checklist:


## Test Plan:

- Enable Rewards
- Dismiss panel and wait 3s for "My First Ad" to appear. 
- Click my first ad, verify that it loads the correct website
- Visit a page, wait 2m
- Click on the ad that shows up and verify it opens the web page

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
